internal/handlers/telegram/handler.go

```diff
@@ -26,3 +26,17 @@ func messageHandler(tg *Client, u tgbotapi.Update) {
 
 	tg.sendToIrc(formatted)
 }
+
+/*
+documentHandler once a document received, uses information about the document
+to format a message to return to the client.
+*/
+func documentHandler(tg *Client, u tgbotapi.Update) {
+	header := tg.Settings.Prefix + u.Message.From.UserName +
```

Comment:
When sending over things like documents and photos, a user isn't strictly saying the message, so we don't need the message prefix and suffix. Here's what the message format looks like on NodeJS:
```
T​jzabel shared a file (image/jpeg) on Telegram with caption: 'my file'
```

Reply:
Prefix and suffix removed.